finalPrep/modZeroHero.js

```diff
@@ -1,52 +1,85 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 // Declare two variables - heroName AND specialAbility - set to strings
+var heroName = 'Spider Man';
+var specialAbility = 'Spidey Sense';
 
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
+var greeting = "What's up? I'm " + heroName + "!";
+var catchphrase = (`Just your friendly neighborhood ${heroName} with ${specialAbility}!`);
```

Comment:
You actually don't need to use the `(  )` around this variable value - you can simply keep it as is with the backticks!